src/components/organisms/AssetActions/Pool/Remove.tsx

```diff
@@ -149,13 +156,41 @@ export default function Remove({
     totalPoolTokens
   ])
 
+  async function computeOceansRemoved(amountPoolShares: string) {
+    const oceanAmount = await ocean.pool.getOceanRemovedforPoolShares(
+      poolAddress,
+      amountPoolShares
+    )
+    setAmountOcean(oceanAmount)
+  }
+
+  function computeOutput() {
+    if (!isAdvanced) {
+      const maxShares =
+        (Number(amountPoolShares) * (100 + Number(slippage))) / 100
+      setMaxShares(`${maxShares}`)
+    } else {
+      const minOceanAmount =
+        (Number(amountOcean) * (100 - Number(slippage))) / 100
+      const minDatatokenAmount =
+        (Number(amountDatatoken) * (100 - Number(slippage))) / 100
+      setMinOceanAmount(`${minOceanAmount}`)
+      setMinDatatokenAmount(`${minDatatokenAmount}`)
+    }
+  }
+
+  useEffect(() => {
+    computeOutput()
```

Comment:
the contents of this method should be within this `useEffect` which is the most simple way to prevent the need to wrap the method in a `useCallback` with it's own dependencies. Otherwise state could get out of sync, or appear not updated.

This will then also prevent this hooks dependency warning: 
<img width="633" alt="Screen Shot 2021-03-23 at 15 14 55" src="https://user-images.githubusercontent.com/90316/112160522-88489b80-8bea-11eb-8f3e-5b3d102c0579.png">